### PR TITLE
Add sdk-features trigger to PRs

### PR DESCRIPTION
**Pull request by Sushisource**
DRAFT while building

Add sdk-features coverage to PRs

### CODE REVIEW

File: .github/workflows/sdk-features-integration.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+name: SDK Features Integration
+
+on: # rebuild any PRs and main branch changes
+  pull_request:
+  push:
+    branches:
+      - main
```
Comment (yiminc) [inline .github/workflows/sdk-features-integration.yml:7]: replace main to master

**Review by yiminc — COMMENTED** [review]
Do we know how long does it take for these tests to run?

File: .github/workflows/sdk-features-integration.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    uses: temporalio/docker-builds/.github/workflows/docker-build-only.yml@just-build-image-workflow
+    with:
+      temporal-server-repo-ref: ${{github.event.pull_request.head.ref}}
+      docker-builds-repo-ref: just-build-image-workflow
+
+  feature-tests-ts:
+    needs: build-docker-image
+    uses: temporalio/sdk-features/.github/workflows/typescript.yaml@main
+    with:
+      version: 1.4.4
```
Comment (yiminc) [inline .github/workflows/sdk-features-integration.yml:25]: is there @latest release tag that we can use? so we don't have to update for every release

Comment (Sushisource) [reply .github/workflows/sdk-features-integration.yml:26]: We're going to add this to the sdk-features side.

Comment (yiminc) [reply .github/workflows/sdk-features-integration.yml:26]: what is this 1.4.4 version?

Comment (Sushisource) [reply .github/workflows/sdk-features-integration.yml:26]: The typescript SDK version

File: .github/workflows/sdk-features-integration.yml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+on: # rebuild any PRs and main branch changes
+  pull_request:
+  push:
+    branches:
+      - master
+
+concurrency: # Auto-cancel existing runs in the PR when a new commit is pushed
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
```
Comment (mindaugasrukas) [inline .github/workflows/sdk-features-integration.yml:11]: I think need a fallback scenario for `main` builds: https://docs.github.com/en/actions/using-jobs/using-concurrency#example-using-a-fallback-value.

```
${{ github.head_ref || github.run_id }}
```

Comment (Sushisource) [reply .github/workflows/sdk-features-integration.yml:11]: Yeah, it does, thanks.

**Review by mindaugasrukas — APPROVED** [review]

**Review by alexshtin — APPROVED** [review]

**Comment by Sushisource** [comment]
Will wait to merge this until we un-private the docker repo for sdk-features since it breaks forks.

**Pull request merged**